sys/include/net/ng_ipv6/addr.h

```diff
@@ -49,7 +49,7 @@ extern "C" {
 /**
  * @brief Data type to represent an IPv6 address.
  */
-typedef union __attribute__((packed)) {
+typedef union {
     uint8_t u8[16];             /**< devided by 16 8-bit words. */
```

Comment:
could you correct the speling errors while you're touching this file?
s/devided/divided/g